sdk/oracle-service/src/implementation/coingecko/CoingeckoOracleProvider.ts

```diff
@@ -175,6 +175,8 @@ export class CoingeckoOracleProvider
 
       const [, price] = Object.entries(responseData)[0]
 
+      const priceValue = price['usd'] ?? price
```

Comment:
should be price[quoteCurrency.toLowerCase()] as specific fiat currecy can be passed as arg